_data/meltano/utilities/notebook/matatika.yml

```diff
@@ -0,0 +1,48 @@
+description: Jupyter Notebook runner and converter
+name: notebook
+label: Jupyter Notebook
+namespace: notebook
+pip_url: git+https://github.com/Matatika/utility-notebook.git
+maintenance_status: development
+repo: https://github.com/Matatika/utility-notebook
+executable: notebook
+logo_url: /assets/logos/utilities/notebook.png
+variant: matatika
+commands:
+  install:
+    args: install $NOTEBOOK_REQUIREMENTS_PATH
+    description: Install the dependencies of a notebook to the plugin's Meltano-managed virtual environment (usually `.meltano/utilities/notebook/venv/`).
+  run:
+    args: run $NOTEBOOK_PATH
+    description: Run a notebook.
+  convert:
+    args: convert $NOTEBOOK_PATH -f $NOTEBOOK_FORMAT -c $NOTEBOOK_CONFIG
+    description: Convert a notebook to another format.
+  run-convert:
+    args: run $NOTEBOOK_PATH -f $NOTEBOOK_FORMAT -c $NOTEBOOK_CONFIG
+    description: Run and convert a notebook.
+settings:
```

Comment:
@ReubenFrankel this looks great!

Are any of these settings required? I don't see any `settings_group_validation`. It seems like you'd need a path in order to do anything. The others look like theyre required depending on the command.

Reply:
Not at any one time, since `install` does not require a path.

Reply:
@ReubenFrankel ahh ok yeah that makes sense 👍 .